textdb/textdb-exp/src/test/java/edu/uci/ics/textdb/exp/twitterfeed/TwitterFeedTest.java

```diff
@@ -0,0 +1,146 @@
+    private final int timeOut = 10;
+    BlockingQueue<String> queue = new LinkedBlockingQueue<String>(100);
+
+    @BeforeClass
+    public static void prepareTuples() throws Exception {
+        exactResults = TwitterFeedTestHelper.getQueryResults(query, NEWYORK, language, limit);
+    }
+
+
+    @Test
+    public void testTwitterFeedlimit() throws Exception {
+        Assert.assertEquals(exactResults.size(), limit);
+
+    }
+
+
+    @Test
+    public void testKeywordQuery() throws Exception {
+
+        List<String> attribute = new ArrayList<>(Arrays.asList("text", "user_screen_name"));
+        Assert.assertTrue(TwitterFeedTestHelper.containsFuzzyQuery(exactResults, query, attribute));
+
+    }
+
+    @Test
+    public void testLocaton() throws Exception {
+        Assert.assertTrue(TwitterFeedTestHelper.inLocation(exactResults, NEWYORK));
+    }
+
+
+    @Before
+    public void setUp() throws Exception {
+        List<String> keyWord = new ArrayList<>(Arrays.asList("is"));
```

Comment:
Introduce a variable, `dummyKeyword` with a value "is".

"keyWord" -> "keywordList"